vendordeps/Phoenix.json

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
We needed to add this external vendor library or the code will not compile. Basically looked on google for "phoenix talon fx java frc" There's a lot of subjects but the one we needed was this:

https://docs.ctre-phoenix.com/en/latest/ch05a_CppJava.html#frc-c-java-add-phoenix

So I visited https://maven.ctr-electronics.com/release/com/ctre/phoenix/Phoenix-latest.json and just copy pasted the content to Phoenix.json like we do for FalconRecharged2021 code.